src/components/dialogs/new_account.tsx

```diff
@@ -20,7 +21,10 @@ const NewAccountDialog: React.FC<NewAccountDialogProperties> = ({isOpen, onReque
 	const {t} = useTranslation();
 
 	const handleConfirmClick = () => {
-		delete_local_for_recovery('').then(() => {
+		console.log('Creating new account...');
+		console.log('Deleting local data...');
+		console.log(password);
```

Comment:
Remove this @dev-blc , My bad

Reply:
Removed
![image](https://github.com/AvailX/avail-wallet/assets/106152383/80eed537-24a9-4ba8-af05-e22a595104b1)